src/components/Editor/SearchBar.js

```diff
@@ -133,19 +135,21 @@ const SearchBar = React.createClass({
     return findDOMNode(this).querySelector("input");
   },
 
-  doSearch(query) {
+  doSearch(query: string) {
     const sourceText = this.props.sourceText;
     const count = countMatches(query, sourceText.get("text"));
     // eslint-disable-next-line react/no-did-update-set-state
     this.setState({ query, count, index: 0 });
     this.search(query);
   },
 
-  onChange(e) {
-    this.doSearch(e.target.value);
+  onChange(e: any) {
```

Comment:
I thought you just need guard against the target value being null.

`if (e.target && e.target.value)`